content/posts/2021/release-0.8.0.md

```diff
@@ -0,0 +1,363 @@
+
+## What's new for the administrators?
+
+### CPU and Disk quota accounting
+
+REANA 0.8 release series allows cluster administrators to set certain
+CPU and Disk quota usage limits for individual users. This can be set
+by Helm configuration option `quota.enabled`.
+
+Please note that if you do enable CPU and Disk quota monitoring, you
+may want to further configure `quota.termination_update_policy` to
+decide whether the CPU and Disk quota usage consumption for users will
+be calculated immediately after workflow terminates or via a periodic
+cron jobs.
+
+### Improved cluster performance
+
+The cluster performance when running numerous concurrent workflows was
+considerably improved. We have tested running up to thousand of
+concurrent workflows running on clusters of up to three hundred
+nodes. There are new Helm configuration options that allow to
+fine-tune REANA cluster components in these conditions, such as:
+
+- ``reana_server.environment.REANA_WORKFLOW_SCHEDULING_POLICY`` allows
+  to set workflow scheduling policy. The value "fifo", meaning
+  "first-in first-out", will execute workflows as they are coming. The
+  value "balanced" will look at how many workflows a particular user
+  launches. If user A uses system heavily, and user B submits new
+  workflows, the system will prefer to schedule the workflow from user
+  B before continuing with workflows of the user A. The "balanced"
+  scheduling policy will also look at the workflow DAG complexity. If
+  a new workflow C would like to launch 10 jobs at the start, and the
+  workflow D only 2 jobs, and the cluster is busy and the current
+  capacity cannot accept more than 4 jobs at the given time, the
+  workflow D would be preferred over workflow C.
+
+- ``reana_server.environment.REANA_RATELIMIT_GUEST_USER`` and
+  ``reana_server.environment.REANA_RATELIMIT_AUTHENTICATED_USER``
+  allow to set REST API rate limit values.  If your cluster is very
+  busy, you can use values such as "2000 per second" enabling heavy
+  traffic from running a high amount of concurrent workflows.
+
+- ``reana_server.environment.REANA_SCHEDULER_REQUEUE_SLEEP`` allows to
+  set sleep time between processing queued workflows. The default
+  value of 15 seconds may be too conservative, as the scheduler will
+  pause for 15 seconds before looking at the incoming workflow
+  queue. If you are running a lot of concurrent workflows, you may
+  want to lower it to 1 second or even 0.1 seconds.
+
+- ``reana_workflow_controller.environment.REANA_JOB_STATUS_CONSUMER_PREFETCH_COUNT``
+  allows tweaking the Rabbit MQ prefetch count for the job status
+  consumer. The default value was tested to behave well up to a
+  thousand of concurrent workflows, so should be probably fine in your
+  setups.
+
+### How to upgrade existing REANA 0.7 clusters
+
+If you are a REANA cluster administrator and you would like to upgrade
+from REANA 0.7.x to REANA 0.8.0, please follow the steps described
```

Comment:
I think in [other places](https://github.com/reanahub/docs.reana.io/pull/97/files#diff-e488cefb8c1614a7fc19bb433c9e9b4785455637355fcd9250220af8b5c0e800R5) we use '*' instead of 'x'.
```suggestion
from REANA 0.7.* to REANA 0.8.0, please follow the steps described
```